src/main/java/frc/robot/Robot.java

```diff
@@ -319,38 +319,6 @@ public void RobotLoop() {
     turretLoop();
 
     driveTrainLoop();
-
-    if (Config.getInstance().getBoolean(Key.ROBOT__HAS_LEDS)) {
-      mBallIndicator.checkTimer();
-    }
-
-    // Climb
-    if (mOperatorInterface.getClimb()) {
-      // climb!
-    }
-
-    // Operator Controls
-    if (mOperatorInterface.getTurretAdjustLeft()) {
```

Comment:
Manual Turret adjustment (Left and Right) can stay

Reply:
Fixed

Reply:
Fixed

Reply:
Fixed